javascript/brahma-firelight/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "type": "module",
   "dependencies": {
-    "brahma-firelight": "^1.5.2"
+    "brahma-firelight": "1.0.6"
```

Comment:
I don't understand the last stable release is 1.5
https://github.com/Shyam20001/rsjs/releases/tag/1.5.0